components/avatar/avatar.vue

```diff
@@ -220,6 +227,13 @@ export default {
       ];
     },
 
+    overlayClasses () {
+      return [
+        'd-bgc-black-900 d-o70 d-ps-absolute d-w100p d-h100p d-d-flex d-ai-center d-zi-base',
```

Comment:
```suggestion
        'd-bgc-black-900 d-o70 d-ps-absolute d-w100p d-h100p d-d-flex d-ai-center d-zi-base1 d-bar-circle',
```

The z-index needs to be 'd-zi-base1' in order to be able to override the avatarImage in certain cases.
Also we need d-bar-circle here so the overlay is always in circle shape

Before
<img width="437" alt="Screenshot 2023-04-13 at 12 24 05 p m" src="https://user-images.githubusercontent.com/87546543/231849790-82795ef0-d621-499c-a976-a6c21d474880.png">

After
<img width="427" alt="Screenshot 2023-04-13 at 12 24 27 p m" src="https://user-images.githubusercontent.com/87546543/231849851-b9e022d3-9be9-45a2-a8e1-4270a135d1de.png">

Reply:
> The z-index needs to be `d-zi-base1` in order to be able to override the avatarImage in certain cases.

Yeah, I thought as you before. However, we may face to multiple admins listen-in case later. If we use `d-zi-base1`, the overlay icon will cover next avatar.
<img width="453" alt="Screen Shot 2023-04-14 at 8 50 18 AM" src="https://user-images.githubusercontent.com/86271164/232094815-806d523e-cca7-486c-ae4b-0cbad63d8974.png">